homeassistant/components/elmo_alarm/__init__.py

```diff
@@ -0,0 +1,232 @@
+"""Support for e-connect Elmo alarm system."""
+from collections import namedtuple
+from datetime import timedelta
+from urllib3.exceptions import HTTPError
+
+import logging
+
+from elmo.api.client import ElmoClient
+from elmo.api.exceptions import PermissionDenied
+import voluptuous as vol
+
+from homeassistant.const import (
+    CONF_HOST,
+    CONF_USERNAME,
+    CONF_PASSWORD,
+    CONF_NAME,
+    CONF_SCAN_INTERVAL,
+    STATE_ALARM_ARMED_AWAY,
+    STATE_ALARM_ARMED_CUSTOM_BYPASS,
+    STATE_ALARM_ARMED_HOME,
+    STATE_ALARM_ARMED_NIGHT,
+    STATE_ALARM_DISARMED,
+)
+from homeassistant.helpers import config_validation as cv
+from homeassistant.helpers.discovery import async_load_platform
+from homeassistant.helpers.dispatcher import async_dispatcher_send
+from homeassistant.helpers.event import async_call_later
+
+_LOGGER = logging.getLogger(__name__)
+
+DOMAIN = "elmo_alarm"
+
+CONF_VENDOR = "vendor"
+
+CONF_STATES = "states"
+CONF_NAME = CONF_NAME
```

Comment:
Remove this new constant `CONF_NAME` and only use the imported one `CONF_NAME`.

Reply:
Sorry it was not intended.